wsd/LOOLWSD.cpp

```diff
@@ -1127,7 +1127,7 @@ void LOOLWSD::innerInitialize(Application& self)
 #ifdef ENABLE_FEATURE_RESTRICTION
             { "restricted_commands", "" },
 #endif
-            { "user_interface.mode", USER_INTERFACE_MODE },
+            { "user_interface.mode", "classic" },
```

Comment:
I would keep USER_INTERFACE_MODE here, because in ios/config.h we still define classic. And in Linux case, we want the new default, don't we?

Reply:
As we have discussed, now USER_INTERFACE_MODE is completely removed.